prow/config/tide.go

```diff
@@ -140,6 +155,22 @@ func (t *Tide) MergeMethod(org, repo string) github.PullRequestMergeType {
 	return v
 }
 
+// MergeCommitTemplate returns a struct with Go template string(s) or nil
+func (t *Tide) MergeCommitTemplate(org, repo string) TideMergeCommitTemplate {
+	name := org + "/" + repo
+
+	v, ok := t.MergeTemplate[name]
+	if !ok {
+		if ov, found := t.MergeTemplate[org]; found {
+			return ov
+		}
+
+		return TideMergeCommitTemplate{}
```

Comment:
nit: The conditional block could be simplified to:
```go
return t.MergeTemplate[org]
```

Reply:
Could you please explain why this works?

Reply:
When you index into a map with a key that does not exist in it:

```go
item, exists: myMap["somekey"]
```

`if !exists` will mean that `item` is the zero value for the type, in this case `TideMergeCommitTemplate{}`